wd_core.au3

```diff
@@ -1507,7 +1507,7 @@ Func __WD_MessageCreator($sFuncName, $sURL, ByRef $sResponseText, $sData = '')
 	Switch $_WD_DEBUG
 		Case $_WD_DEBUG_Full ; in case of $_WD_DEBUG_Full  >  Full $sResponseText
 			__WD_ConsoleWrite($sFuncName & ": URL=" & $sURL & "; $sData=" & $sData)
-			If $_WD_RESPONSE_TRIM <> -1 Then
+			If $_WD_RESPONSE_TRIM <> -1 And StringLen($sResponseText) > $_WD_RESPONSE_TRIM Then
```

Comment:
maybe easier:
```autoit
If StringLen($sResponseText) > $_WD_RESPONSE_TRIM Then
```

Reply:
Unless I'm missing something, I don't see how that will work correctly when  `$_WD_RESPONSE_TRIM = -1 `

Reply:
My fault